beherabikram

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Sara Jaoude",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"
+  "displayStatsPublic": true,
```

Comment:
This tag is no longer needed, consider removing it.
![nolongern](https://github.com/EddieHubCommunity/LinkFree/assets/19821445/766faa33-2b03-4eb9-a792-d832ffb175e6)
```suggestion
```